packages/functional-tests/tests/react-conversion/forceAuth.spec.ts

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { test, expect } from '../../lib/fixtures/standard';
+import { getReactFeatureFlagUrl } from '../../lib/react-flag';
+
+test.describe('force auth react', () => {
+  test.beforeEach(async ({ pages: { configPage } }) => {
+    test.slow();
+    // Ensure that the feature flag is enabled
+    const config = await configPage.getConfig();
+    test.skip(config.showReactApp.signInRoutes !== true);
+    test.skip(config.showReactApp.signUpRoutes !== true);
+  });
+
+  test('redirects to signin with registered email', async ({
```

Comment:
Nit: maybe instead of `redirects`, use the word `displays` or something like it, since we don't actually perform a redirect in this case?